File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.py
import sys
input = sys.stdin.readline

def factorial(num):
    result = 1
    for i in range(1, num + 1):
        result = result * i
    return result

N, K = map(int, input().split())

print(int(factorial(N) / (factorial(N-K) * factorial(K))))

File: 백준/Bronze/11050. 이항 계수 1/이항 계수 1.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split(" "); // 이런 방식으로 할 수 있더군요..!
        int n = Integer.parseInt(input[0]);
        int k = Integer.parseInt(input[1]);

        System.out.println(factorial(n) / (factorial(k) * factorial(n - k)));
    }

    // 그냥 입력이 작아서 아래 함수로 처리했습니다.
    static int factorial(int num) {
        if (num == 0 || num == 1) return 1;
        return num * factorial(num - 1);
    }
}